sdk/android/test/sdk.testapp/src/main/java/com/microsoft/windowsazure/mobileservices/sdk/testapp/test/MobileServiceSyncTableTests.java

```diff
@@ -719,11 +723,394 @@ public void testDeleteCancelsAllWhenInsertIsInQueue() throws Throwable {
         assertEquals(client.getSyncContext().getPendingOperations(), 1);
```

Comment:
name of this function is suggests it returns operations not operationCount,
Name should be getPendingOperationsCount or something similar